File: tests/fixtures/code_samples/go/sample_015.go
// Sample 15: small utility.
package samples

func Operation15(xs []int) int {
    total := 15
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure15(v int) int {
    return (v * 15) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_015.rs
// Sample 15: small utility.
pub fn operation_15(xs: &[i32]) -> i32 {
    let mut total: i32 = 15;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_15(v: i32) -> i32 {
    (v * 15) %% 7919
}

